### PR TITLE
mgr/dashboard_v2: Update documentation

### PR DESCRIPTION
**Pull request by p-se**
* Nested Controllers
* How to use RESTController

Signed-off-by: Patrick Nawracay <pnawracay@suse.com>

### CODE REVIEW

**Review by rjfd — DISMISSED** [review]
This is very helpful, thank!

File: src/pybind/mgr/dashboard_v2/README.rst [hunk 1]
```diff
@@ -115,14 +115,45 @@ following code::
     def default(self, *args):
       return "Hello"
 
-Reload the Dashboard module and then you can access the above controller from
-the web browser using the URL http://mgr_hostname:8080/api/ping2.
+Every path given in the ``ApiController`` decorator will automatically prefixed
```
Comment (callithea) [inline src/pybind/mgr/dashboard_v2/README.rst]: Maybe "[...] will _be_  automatically prefixed [...]"?

File: src/pybind/mgr/dashboard_v2/README.rst [hunk 2]
```diff
@@ -115,14 +115,45 @@ following code::
...
+with ``api``.  Reload the Dashboard module and then you can access the above
```
Comment (callithea) [inline src/pybind/mgr/dashboard_v2/README.rst]: "_After reloading_ the Dashboard module you can access the above _mentioned_ controller [...]"

**Review by LenzGr — APPROVED** [review]
LGTM, thanks!

**Review by callithea — APPROVED** [review]

**Pull request merged**